app/retail/templates/about.html

```diff
@@ -16,6 +16,39 @@
 {% endcomment %}
 {% load i18n static %}
 <!DOCTYPE html>
+<style type="text/css">
+  div.teamProfileLeft, div.teamProfileRight {
+    background-color: #eff4ee;
+    padding: 1em;
+    margin-bottom: 2em;
+    box-shadow: 0px 2px 5px 0px grey;
```

Comment:
Could we use the hexcode instead to ensure the same color is rendered across all browsers ?